### ISSUE TITLE
UI: Images for sounds

### ISSUE THREAD

**Issue by mizzunet** [ashutoshgngwr/noice#596]
### Is your feature request related to a problem? Please describe.

Not at all
### Describe the solution you'd like

It's a UI suggestion, not necessarily needed but would be so great looking. That is, images for sounds like this Windows program does, Ambie, https://github.com/jenius-apps/ambie
![image](https://user-images.githubusercontent.com/10193999/115304798-02852500-a183-11eb-9a98-bce028da108b.png)

**Comment by PanderMusubi** [comment]
When there is an option to show images, please keep also the possibility to show the current GUI without images.

**Comment by ashutoshgngwr** [comment]
tl;dr No images. Period.

The images do add some visual flair and improve UX significantly, but let's just say I am too lazy to put in the work in something that won't improve Noice's function. If someone can procure royalty-free images for all sounds, I can put in the effort to customize the current UI to be more visually pleasing. One more consideration is using vector graphics since Android device resolutions vary from very small screens to very large screens.

**Comment by mizzunet** [comment]
> tl;dr No images. Period.
> 
> The images do add some visual flair and improve UX significantly, but let's just say I am too lazy to put in the work in something that won't improve Noice's function. If someone can procure royalty-free images for all sounds, I can put in the effort to customize the current UI to be more visually pleasing. One more consideration is using vector graphics since Android device resolutions vary from very small screens to very large screens.

I understand😁

**Comment by Watson15683** [comment]
I also would like that Image support. But i also totally understand that it is a lot of effort, that does not add any value appart form the visual aspect.

But i just want to add that maybe Images could make the frist impression of the app much better, which i think matters to convinve "First Time users" to come back to the app.

And in case someboy would produce Vector Graphics, which **ratios** and **general Style** would be appreciated? (Colored, B and W, more curvy or more edgy)

**Comment by ashutoshgngwr** [comment]
> I also would like that Image support. But i also totally understand that it is a lot of effort, that does not add any value appart form the visual aspect.
> 
> But i just want to add that maybe Images could make the frist impression of the app much better, which i think matters to convinve "First Time users" to come back to the app.
> 
> And in case someboy would produce Vector Graphics, which **ratios** and **general Style** would be appreciated? (Colored, B and W, more curvy or more edgy)

Hi @Watson15683, I was experimenting with the layout and I think I am onto something. After your response, my intention was to quickly setup a dirty prototype so everyone has an exact idea about it. It's gonna take sometime for me to setup the full thing, and I'll let you know as soon as I do.

**Comment by ashutoshgngwr** [comment]
Hey everyone! I have finished restructuring the UI to accommodate app icons. Please see pull request #601. I have written there all necessary instructions to add more icons.
